CHANGELOG.unreleased.md

```diff
@@ -859,6 +859,22 @@ function doSmth() {
 }
 ```
 
+#### TypeScript: Fix incorrectly removes parentheses around inline jsx elements being accessed in tsx file ([#6640] by [@sosukesuzuki])
```

Comment:
Consider changing the wording to "TypeScript: sometimes removing parentheses around JSX made the code unparseable".

Reply:
Thank you, I'm not good at English... It will be helpful if you continue to review.

Reply:
Your English is fine, but "fix ... removes parentheses" sounds like your fix removes them, not fixes their removal.